apps/google-analytics-4/frontend/src/components/config-screen/api-access/display/ChecklistUtils.tsx

```diff
@@ -93,113 +95,134 @@ export type ChecklistStatus = {
+        "For technical reasons we were unable to retrieve the service account key that's stored for this app. If the problem persists, try reinstalling the service account key or contact support."
       ),
       title: CHECKLIST_NAMES.serviceAccount,
-      description: 'Unable to retrieve service account key file.',
+      description:
+        'Unable to retrieve your stored service account key. Re-install it if the problem persists',
       disabled: false,
     },
   },
   AdminApi: {
     success: {
-      icon: getSuccessIcon('Admin API successfully enabled'),
+      icon: getSuccessIcon(''),
       title: CHECKLIST_NAMES.adminApi,
-      description: 'Success!',
+      description: 'Analytics Admin API is enabled',
       disabled: false,
     },
     firstTimeSetup: {
-      icon: getArrowIcon('Please enable the Admin API to run this check'),
+      icon: getArrowIcon(
+        'The Analytics Admin API allows Contentful to fetch the list of properties your service account has access to. Please enable this API inside your Google Cloud project to continue.'
+      ),
       title: CHECKLIST_NAMES.adminApi,
       description: 'Analytics Admin API is not yet enabled',
       disabled: false,
     },
     invalidServiceAccount: {
       icon: getClockIcon(
-        'Service account errors detected, please install a correctly configured service account installation'
+        'This check will not run until a valid service account key has been provided'
       ),
       title: CHECKLIST_NAMES.adminApi,
-      description: 'Awaiting a correctly configured service account installation',
+      description: `Provide a valid service account key to run this check`,
       disabled: true,
     },
     error: {
-      icon: getErrorIcon('Failed to connect to the Admin API - did you enable the api?'),
+      icon: getErrorIcon(
+        'The Analytics Admin API allows Contentful to fetch the list of properties your service account has access to. Please enable this API inside your Google Cloud project.'
+      ),
       title: CHECKLIST_NAMES.adminApi,
-      description: 'Analytics Admin API must be enabled to use this app',
+      description: 'Analytics Admin API is not enabled',
       disabled: false,
     },
   },
   DataApi: {
     success: {
-      icon: getSuccessIcon('The Analytics Data API is enabled'),
+      icon: getSuccessIcon(''),
       title: CHECKLIST_NAMES.dataApi,
-      description: 'Success!',
+      description: 'Analytics Data API is enabled',
       disabled: false,
     },
     firstTimeSetup: {
-      icon: getArrowIcon('Please enable the Data API to run this check'),
+      icon: getArrowIcon(
+        'The Analytics Data API allows Contentful to fetch analytics data from the property you specify. Please enable this API inside your Google Cloud project to continue.'
```

Comment:
a little bit of divergence here from the one above – let's end them both the same: `Please enable this API inside your Google Cloud project.`

Reply:
also i think `in` is better than `inside` (mentioned this below, too)

Reply:
> a little bit of divergence here from the one above – let's end them both the same: Please enable this API inside your Google Cloud project.

Divergence here is intentional. For first time setup we're emphasizing that you need to do this to continue the process. For the second you just need to enable it cuz somehow it got disabled.

Reply:
ah, gotcha, thanks for clarifying. carry on